parameters/creating-and-using-parameters.md

```diff
@@ -0,0 +1,34 @@
+# Creating and Using Parameters
+
+See this [**Workflow Parameters Tutorial**](../example-workflows/using-workflow-parameters.md) notebook for a full walkthrough of the API with concrete examples.
+
+### Creating Parameters
+
+Parameters are identified by their name, and *must* have default values. They can be created explicitly or implicitly.
+
+To create a parameter explicitly, use `client.create_param(<param_name>, default=<default_val>)`. These parameters can be fed into operators just like any other artifacts. 
+
+They can also be referenced within SQL queries using the double-bracket syntax! Eg. A query `select ... where col_name={{country_names}}` is completely valid, as long as a parameter named "country_names" exists.
+If "country_names" has current value `"'Argentina'"`, the query will implicitly expand into `select ... where col_name='Argentina'`. Note that SQL parameters can only be strings,
```

Comment:
This sentence is slightly confusing. I might wonder: If it's a string, why do I have to insert quotes?

I think we want to clarify that the Python value is required to be a string. When doing interpolation, we don't do any type guards like adding quotes.